quickwit/quickwit-cli/src/logger.rs

```diff
@@ -112,6 +113,7 @@ pub fn setup_logging_and_tracing(
             .with_resource(Resource::new([
                 KeyValue::new("service.name", "quickwit"),
                 KeyValue::new("service.version", build_info.version.clone()),
+                KeyValue::new("service.node", node_name),
```

Comment:
```suggestion
                KeyValue::new("service.instance.id", node_id),
```
OTEL strongly discourages custom attributes that overlap with existing semantics like `service.node`:
https://opentelemetry.io/docs/specs/semconv/registry/attributes/service/#service-instance-id